wfdb/plot/plot.py

```diff
@@ -156,10 +280,12 @@ def get_plot_dims(signal, ann_samp):
 
     Parameters
     ----------
-    signal : 1d or 2d numpy array, optional
-        The uniformly sampled signal to be plotted. If signal.ndim is 1, it is
-        assumed to be a one channel signal. If it is 2, axes 0 and 1, must
-        represent time and channel number respectively.
+    signal : 1d or 2d numpy array or list, optional
+        The uniformly sampled signal or signals to be plotted.  If signal
```

Comment:
It looks like you added `signal = _expand_channels(signal)` in `plot_items` right before  `get_plot_dims` is (solely) called. Perhaps `get_plot_dims` should only accept `signal` as a list then?